src/modules/draw/draw-directive.js

```diff
@@ -1,10 +1,11 @@
+require('../util');
```

Comment:
is there a reason you are using `require` instead of `import` here?

Reply:
no. I fixed that.